src/com/owncloud/android/operations/RefreshFolderOperation.java

```diff
@@ -177,12 +166,18 @@ private void updateUserProfile() {
         }
     }
 
-    private void updateCapabilities(){
-        GetCapabilitiesOperarion getCapabilities = new GetCapabilitiesOperarion();
-        RemoteOperationResult  result = getCapabilities.execute(getStorageManager(), mContext);
-        if (!result.isSuccess()){
-            Log_OC.w(TAG, "Update Capabilities unsuccessfully");
+    private OwnCloudVersion updateCapabilities() {
```

Comment:
I renamed the method to 'syncCapabilities()', that is more accurate.

Reading the code of the method I really hope we never need to reuse this in any other place. First, it returns an ```OwnCloudVersion```  instance as a 'side effect' of syncing capabilities, instead of returning capabilities instance, so the name of the method is confusing. Second, it checks the ```AccountManager``` for the version as a fallback for servers that do not respond capabilities (or don't support them), code that I would prefer to get rid of because users just should update to servers with capabilities API instead of staying with elder ones - but I don't trust enough that if we forget that possibility it will not bite us back.

Besides, util classes, the lesser, the better. Static methods are perfect to trigger career conditions in multithreaded environments. Or to dump all kind of stuff in without thinking twice.